src/pyodide/internal/workers.py

```diff
@@ -0,0 +1,76 @@
+# This module defines a Workers API for Python. It is similar to the API provided by
+# JS Workers, but with changes and additions to be more idiomatic to the Python
+# programming language.
+from http import HTTPMethod, HTTPStatus
+from typing import TypedDict, Unpack
+
+import js
+
+import pyodide.http
+from pyodide.http import pyfetch
```

Comment:
I didn't do this because I want the code to be explicit when referring to Pyodide's `FetchResponse`, since I am defining a new class of the same name. Did I miss something?

Reply:
Ah. Then maybe it's nice to do:
```py
from pyodide.http import pyfetch, FetchResponse as OrigFetchResponse
```
but either way.